Makefile

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
I'd rather not complexify the PR with updating the Makefile. Can you remove this and make it in another PR?

Reply:
Removed additions of go.mod, .gitignore and Makefile - will leave for future PR.